core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamlet.scala

```diff
@@ -55,9 +58,20 @@ abstract class AkkaStreamlet extends Streamlet[AkkaStreamletContext] {
 
       if (activateCluster && localMode) {
         val clusterConfig = ConfigFactory.parseResourcesAnySyntax("akka-cluster-local.conf")
-        val fullConfig    = clusterConfig.withFallback(updatedStreamletDefinition.config)
-
-        val system  = ActorSystem(streamletDefinition.streamletRef, ConfigFactory.load(fullConfig))
+        // Add persistence configuration
+        val clusterConfigDB =
```

Comment:
I feel there should be an `activatePersistence` that drives this, should be a choice. Maybe 
`extends AkkaStreamlet with Clustering with Persistence` or something like that?

Reply:
The specific persistence that you want to use could also be driven by the trait, probably. or through cli and config.

Reply:
Thats a good question. For me usage of persistence is an option for clustering. I do not see persistence used without clustering. And the way it is done now, if you use clustering and add persistence configuration, then its clustering with persistence, otherwise its just clustering